doc/material-migration.md

```diff
@@ -8,6 +8,31 @@ uid: Uno.Themes.Material.Migration
 
 The Uno Material v5 packages introduce a new dependency on the [Uno Themes](https://www.nuget.org/packages/Uno.Themes.WinUI) package. Uno Themes is the base library for all design system implementations going forward. As a result, the following changes have been made:
 
+### TextBox `PlaceholderText` and `Header`
```

Comment:
We should move this under a subsection with a title like Upgrading to 5.1 or something

Reply:
Changes pushed.

Reply:
![image](https://github.com/user-attachments/assets/1ba72437-bc8f-484a-aa27-673cffa8fdfa)